database/migrations/1743075304168_add_organization_status.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "student_organizations";
+  protected organizationStatus = ["active", "inactive", "dissolved", "unknown"];
+
+  async up() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table
+        .enum("organization_status", this.organizationStatus, {
+          useNative: true,
+          enumName: "organization_status",
+          existingType: false,
+        })
+        .notNullable();
+    });
+  }
```

Comment:
You'll need to set a default value for this column so that existing data can be migrated properly (at least for the initial column addition, you can delete the default value in a second query)